modules/protocol/resources/META-INF/smithy/smithy4s.smithy

```diff
@@ -14,7 +14,22 @@ namespace smithy4s.api
 @uuidFormat
 string UUID
 
-@protocolDefinition
+@protocolDefinition(traits: [
+    "smithy.api#cors",
```

Comment:
Does @discriminated belong here?

Reply:
Also @ error, if it's a thing (can't remember, on phone, just spitballing)

Reply:
didn't see the previous comments when I was writing this :D

Reply:
No worries :) appreciate the comments!